root/static/scripts/edit/URLCleanup.js

```diff
@@ -892,9 +916,12 @@ const CLEANUPS = {
       if (m) {
         const frBnF = m[1];
         const phbt = '0123456789bcdfghjkmnpqrstvwxz';
-        const controlChar = phbt[Array.from(frBnF).reduce(function (sum, c, i) {
```

Comment:
Removing `function ` here (using an arrow function) would more simply resolve the max-len issue.